lib/Optimizer/Transforms/Mapping.cpp

```diff
@@ -0,0 +1,624 @@
+/*******************************************************************************
+ * Copyright (c) 2022 - 2023 NVIDIA Corporation & Affiliates.                  *
+ * All rights reserved.                                                        *
+ *                                                                             *
+ * This source code and the accompanying materials are made available under    *
+ * the terms of the Apache License 2.0 which accompanies this distribution.    *
+ ******************************************************************************/
+
+#include "cudaq/Optimizer/Dialect/Quake/QuakeOps.h"
+#include "cudaq/Optimizer/Transforms/Passes.h"
+#include "cudaq/Support/Device.h"
+#include "cudaq/Support/Placement.h"
+#include "llvm/ADT/SmallSet.h"
+#include "llvm/ADT/StringSwitch.h"
+#include "llvm/Support/FileSystem.h"
+#include "llvm/Support/ScopedPrinter.h"
+#include "mlir/Dialect/Func/IR/FuncOps.h"
+#include "mlir/Transforms/TopologicalSortUtils.h"
+
+#define DEBUG_TYPE "quantum-mapper"
+
+using namespace cudaq;
+using namespace mlir;
```

Comment:
We should stick with exactly one using namespace. (Helps prevent any odd ambiguities/name collisions that might arise.)

Reply:
> We should stick with exactly one using namespace. (Helps prevent any odd ambiguities/name collisions that might arise.)

Hmm, any ambiguities would trigger compilation errors, so that wouldn't go unnoticed. FWIW, we use multiple namespaces in ~20 other files in our repo, and one could argue that it helps code readability by making the code less verbose, but I will make the requested change in this file.

Reply:
Updated in cd4eda08.